src/main/java/com/xxxx/example/servlet05.java

```diff
@@ -0,0 +1,25 @@
+package com.xxxx.example;
+
+import database_connection.Database;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+@WebServlet("/ser05")
+public class servlet05 extends HttpServlet {
```

Comment:
These class names are a bit obscure and hard to understand, can we re-factor them?

Reply:
Yes, for the latter ones, I named them with the event type.